estate_account/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_state_sold(self):
+        invoice_vals_list = []
+        for record in self:
+            invoice_vals = {
+                'partner_id': record.buyer_id.id,
+                'move_type': 'out_invoice',
+                'journal_id': self.env['account.journal'].search([('type', '=', 'sale')], limit=1).id,
```

Comment:
Why do you need a journal?
And what if there is none defined in the db? Don't you need to create one in that case?